cpp/BUILD.bazel

```diff
@@ -157,6 +157,7 @@ cc_test(
         "counter.so",
         "plus.so",
         "ray_cpp_pkg",
+        "src/ray/test/cluster/ray-test-SNAPSHOT.jar",
```

Comment:
Use "//java:libio_ray_ray_test.jar“ instead, then we don't have to manually prepare the jar file.